packages/augur-ui/src/modules/common/selection.tsx

```diff
@@ -208,12 +208,12 @@ class Dropdown extends Component<DropdownProps, DropdownState> {
         onClick={this.toggleList}
         onMouseEnter={() => {
           // Outcome selector in order ticket open on click, not hover.
-          if (showColor === false) {
+          if (showColor !== true) {
```

Comment:
showColor can be undefined, false and true. Setting ```showColor !== true``` will enable "mouse over on the button triggers the hover state", which is what we expect.